addon/services/page-title-list.js

```diff
@@ -213,6 +216,11 @@ export default class PageTitleListService extends Service {
     return title.join('');
   }
 
+  willDestroy() {
+    super.willDestroy();
+    this.router.off(RouterEvent.ROUTE_DID_CHANGE, this.scheduleTitleUpdate);
```

Comment:
In real life application service does not get destroyed during application session but in acceptance tests it is nice to clean up properly.